types/ChangedFiles.js

```diff
@@ -12,6 +12,7 @@ import type {Path} from 'types/Config';
 export type Options = {|
   lastCommit?: boolean,
   withAncestor?: boolean,
+  sinceCommit?: ?string,
```

Comment:
`?` on both sides, does that mean anything? Ref https://github.com/facebook/jest/pull/5176#discussion_r158800238